### PR TITLE
Added current users

### LINKED ISSUE #161: Add live examples of Paper Badger to the README
Two publishers are using Paper Badger in production! ORCID also displays badges! Add links to these in the README so people can see how it works live.

Examples listed in this blog post:
https://www.mozillascience.org/contributorship-badges-for-science-view-them-now

### PR DESCRIPTION
**Pull request by jinankjain**
Fixes #161 

@acabunoc have a look at this one.

### CODE REVIEW

**Comment by acabunoc** [comment]
Thanks @jinankjain! This is better than I expected!

A few suggestions:

1. Could you add links to specific papers where people can see this? I know it's only live on a few papers right now, so pointing directly to them would help :) 
2. Can you move this up to the end of the 'Using Paper Badger' section -- as a smaller subsection
3. Rename the section to just 'Current Users'.
4. Remove the other screenshot in the 'Using Paper Badger' section. One screenshot is probably enough, and I like yours!

This is really helpful, thank you again!

**Comment by jinankjain** [comment]
@acabunoc Any reviews on this one?

File: README.md [hunk 1]
```diff
@@ -12,10 +12,18 @@ You can display contributorship badges for science on your own site! Researchers
...
 
-![Badge Preview](./public/img/badge_preview.jpg)
-
 You can either use paper-badger-widget.js ([documentation](docs/paper-badger-widget.md)), which supports old browsers or widget.js ([documentation](docs/widget.md)), which is in development and currently only supports evergreen browsers and IE9+.
 
+### Current Users
+
+Two journals, [GigaScience (BioMed Central)](http://gigascience.biomedcentral.com/) and [Journal of Open Research Software (Ubiquity Press)](http://openresearchsoftware.metajnl.com/) have added the Paper Badger widget to their papers. 
+
+This a [link](http://openresearchsoftware.metajnl.com/articles/10.5334/jors.ad/metrics/#author-contributions) to one of the paper where you could find it live.
```
Comment (acabunoc) [inline README.md]: Small tweak! Can you change this line to include the paper title like below:

~~~md
JORS example: [A Web-based modeling tool for the SEMAT Essence theory of software engineering](http://openresearchsoftware.metajnl.com/articles/10.5334/jors.ad/metrics/#author-contributions)

File: README.md [hunk 2]
```diff
@@ -12,10 +12,18 @@ You can display contributorship badges for science on your own site! Researchers
...
+
+The featured image is an example of how badges will appear on a paper.
```
Comment (acabunoc) [inline README.md]: I think you can delete this line :)

**Comment by acabunoc** [comment]
I left a couple inline comments. Almost there! This is really great, thanks again!

**Comment by jinankjain** [comment]
@acabunoc  I had made the necessary changes :)

**Comment by acabunoc** [comment]
Perfect!

**Pull request merged**